soc/arm64/nxp_imx/imx8m/soc.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef SOC_H
+#define SOC_H
+
+#include <sys/util.h>
+#include <toolchain.h>
+
+#ifndef _ASMLANGUAGE
+#include <devicetree.h>
+#endif
```

Comment:
this is not required, ref https://github.com/zephyrproject-rtos/zephyr/pull/41544